src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -1284,6 +1284,15 @@ ApplicationManagerImpl::GetHandshakeContext(uint32_t key) const {
   return SSLContext::HandshakeContext();
 }
 
+bool ApplicationManagerImpl::IsNaviApp(const int32_t& session_key) const {
+  ApplicationSharedPtr app = application(session_key);
```

Comment:
@okozlovlux It is possible both to be used, look at ApplicationManagerImpl::CanStartProtectedService()